yamlReader.py

```diff
@@ -0,0 +1,167 @@
+import yaml
```

Comment:
This code is overly inelegantly written.

You could deserialise everything from that YAML file in one line:
```python
yamlData = yaml.safe_load(open(start.yamlName()))
```
This would return a dictionary, so

```
>>> data['default_date_day']
10
>>> data.get('default_date_day')
10
>>> data['employees'][0]['employee']['name']
'Leif'
>>> data['employees'][1]['employee']['hourly_wage']
180
>>> 
```

Reply:
Implemented with safe load and 2 global variables